scripts/version_mapper.py

```diff
@@ -105,13 +105,14 @@ def update_switch_version_file(
     # Include the announcement in all available release folders. Note that
     # these are still accessible even if they are not included in the dropdown.
     old_release_folders = []
-    for path in Path("release").iterdir():
-        if path.is_dir() and path.name != latest_stable_version:
+    ignore_folders = ["_static", "_sources", latest_stable_version]
```

Comment:
```suggestion
    ignore_folders = ["_static", "_sources", latest_stable_version]
```

I think you might want to ignore any folder that starts with an underscore... Those two are the typical ones but there are some docs that have more:
![image](https://user-images.githubusercontent.com/37798125/204479234-436c34bc-be8c-4b63-b351-2746df446c35.png)